src/Downloader.ts

```diff
@@ -278,54 +266,8 @@ function parseHash(hash: string): any {
         }, {});
 }
 
-async function checkForCrashClusters(page: Page): Promise<number> {
-    await sleep(1500).then(() => console.log('Hello after first sleep'));
-    await page.waitFor(4770);
-
-    // TODO waiting for .gwt-Label is not definitive as many pages have these 
-    // CSS selectors, so what's unique yet consistently identifiable AND 
-    // available on this page?
-    await page.waitForSelector('.gwt-Label');
-
-    await sleep(900).then(() => console.log('Hello after second sleep'));
-    await page.waitFor(1004);
-
-    var clusters = 0;
-    const labels = await page.$$eval('.gwt-Label', (el: any[]) => el.map(el => el.textContent));
-    const realTimeCrashesText = labels.filter(textContent => textContent.includes("Real-time crashes"));
-    const screenshotFilename = `realtime-crashes-screenshot_${Date.now()}.png`;
-    if (realTimeCrashesText.length == 0) {
-        console.log("Warning, 'Real-time crashes' message not found, is something wrong? see " + screenshotFilename);
-        await page.screenshot({ path: screenshotFilename, fullPage: true });
-
-    } else {
-        // We're on the right page, hopefully... Now try to get the count. 
-        // The count is only provided when there are 0 crash clusters or less than a page worth.
-        console.log("Info, 'Real-time crashes' message found, see " + screenshotFilename);
-        await page.screenshot({ path: screenshotFilename, fullPage: true });
-
-        const clusterText = labels.filter(textContent => textContent.includes("crash clusters")).toString();
-
-        if (clusterText.length > 0) {
-            clusters = parseInt(clusterText);
-            console.log("Found [" + clusters + "] crash clusters");
-        } else {
-            console.log("No count available, assuming at least a page of crash clusters.")
-            // Could we also save the contents of the page for post-hoc analysis? I've seen 1 crash cluster
-            // shown in  realtime-crashes-screenshot_1572475282166.png yet our code didn't find a match.
-            clusters = -1;  // We assume there are plenty but don't know for sure.
-        }
-    }
-
-    // const publishedPackages = availablePackages.filter(p => p.status == "Published");
-    // const gwtLabels = await page.$$eval('[.gwt-Label]]', (as: any[]) => as.map());
-    // els => els.forEach((el: any) => el.click()));
-    // const crashUrl: string = await row.$eval('.related-link', (el: any) => el.href) as any;
-    return clusters;
-}
-
 async function getCrashClusterIds(page: Page): Promise<string[]> {
-    await page.waitForSelector('[data-type="errorLocation"]'); // loading
+    await page.waitForSelector('[data-type="errorLocation"],section[role=article] > p:not([aria-hidden])'); // loading
```

Comment:
This fixes the waiting issue when there are no crashes

Reply:
Thank for this change to help catch the relevant behaviour. BTW I do find it counter-intuitive that we end up waiting for seeming irrelevant elements, especially as I reckon it'll cause maintenance issues in future as the page contents change. I'll write up some more thoughts in a new issue in the hope that we can make the code as easy as practical to maintain in future.

Reply:
This element is the paragraph that says something to the effect of "there are no crashes", so I'd say it's not an irrelevant element ;)

Reply:
I agree, it's relevant, hence I said 'seemingly irrelevant' :) If we asked another developer to explain what this code is waiting for I doubt they'd be able to explain. And if (when) the implementation of the page changes then working out which this element was relevant would be a challenge for anyone except those who were involved in working out why it was relevant.  As I mentioned elsewhere I'll add some thoughts in a new issue for the project.